themes/book/app/root.tsx

```diff
@@ -10,14 +10,19 @@ import {
   getThemeSession,
   ContentReload,
   SkipTo,
+  renderers as defaultRenderers,
 } from '@myst-theme/site';
 export { AppErrorBoundary as ErrorBoundary } from '@myst-theme/site';
 import { createSearch as createMiniSearch } from '@myst-theme/search-minisearch';
 import { Outlet, useLoaderData } from '@remix-run/react';
-import { SearchFactoryProvider } from '@myst-theme/providers';
+import { SearchFactoryProvider, mergeRenderers } from '@myst-theme/providers';
+import type { NodeRenderers } from '@myst-theme/providers';
 import type { ISearch, MystSearchIndex } from '@myst-theme/search';
 import { SEARCH_ATTRIBUTES_ORDERED } from '@myst-theme/search';
-import { useCallback, useMemo } from 'react';
+import { BLOCK_RENDERERS } from '@myst-theme/jupyter';
```

Comment:
Can we call these `JUPTYER_RENDERERS`, I suspect we will need to change the `inlineOutputs` in the near future, so they won't all be blocks.

Reply:
Sure! It might be a bit weird that we also have the other renderers like output and embed, but we can figure that out later.

Reply:
@rowanc1 I've exported the old `OUTPUT_RENDERERS` so that the `site` package continues to pull in only the output renderers, and not the new block renderer. We should look to deprecate that in future (probably just a breaking change).